Flu-Vacc/plot_output.py

```
@@ -6,7 +6,7 @@
 import plotly.io as pio
 from plotly.subplots import make_subplots
 
-from epx import Job
+from epx import FREDJob
```

Ruff (0.11.9)

`epx.FREDJob` imported but unused

Remove unused import: `epx.FREDJob`

(F401)